apps/monitoring-cf.yml

```diff
@@ -0,0 +1,51 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  StepFunctionArn:
+    Type: String
+
+  ApiId:
+    Type: String
+
+Resources:
+
+  AlarmTopic:
+    Type: AWS::SNS::Topic
+
+  StepFunctionAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions:
+        - !Ref AlarmTopic
+      AlarmDescription: Hyp3 processing failures
+      ComparisonOperator: GreaterThanThreshold
+      Period: 60
+      EvaluationPeriods: 1
+      Threshold: 0
+      Dimensions:
+        - Name: Processing Failures
```

Comment:
`Name` is the AWS-defined dimension to filter by, I believe `StateMachineArn` in this case